libs/expo/betterangels/src/lib/screens/Clients/index.tsx

```diff
@@ -35,7 +35,7 @@ export default function Clients({ Logo }: { Logo: ElementType }) {
   const [createNote] = useCreateNoteMutation();
   const [offset, setOffset] = useState<number>(0);
```

Comment:
> We can remove the local offset state and rely on pageInfo.offset from the response. This way, we always use the server’s knowledge of the current offset rather than maintaining our own copy.
https://github.com/BetterAngelsLA/monorepo/pull/822#discussion_r1915416761